<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropperClass</class>
 <widget class="QWidget" name="DropperClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropper</string>
  </property>
  <widget class="GradientSlider" name="test_1" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="GradientSlider" name="test_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>31</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="PaletteHSV" name="test_3" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>561</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorPreview" name="test_4" native="true">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>80</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GradientSlider</class>
   <extends>QWidget</extends>
   <header>ColorPicker/GradientSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaletteHSV</class>
   <extends>QWidget</extends>
   <header>ColorPicker/PaletteHSV.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPreview</class>
   <extends>QWidget</extends>
   <header>ColorPicker/ColorPreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="dropper.qrc"/>
 </resources>
 <connections/>
</ui>
